src/civis/response.py

```diff
@@ -90,10 +90,11 @@ def convert_response_data_type(
             data = response
 
         if isinstance(data, list):
-            return [
+            responses = [
                 Response(d, headers=headers, from_json_values=from_json_values)
                 for d in data
             ]
+            return ListResponse(responses, headers)
```

Comment:
Defining a `ListResponse` object with headers info is the key to allowing the `job_logs` function to use the regular snake-case `client`, resolving an issue that Mike highlighted [here](https://github.com/civisanalytics/civis-python/pull/509#discussion_r1966209451).

Reply:
Nice!